bokehjs/test/index.coffee

```diff
@@ -30,3 +30,7 @@ blacklist.push('constructor')
 for own key, val of global.window
   if blacklist.indexOf(key) == -1
     global[key] = val
+
+global.HTMLElement = global.window.HTMLElement
+global.Event = global.window.Event
+global.Image = global.window.Image
```

Comment:
The last example from that wiki may work for us, but we would need to manually compile scripts (coffeescript) to make it work. Now it's all automated given we use `require()` and on-the-fly compilation.